### ISSUE TITLE
Add support for detecting gradle kotlin dsl files

### ISSUE THREAD

**Issue by jaro2gw** [heroku/heroku-buildpack-gradle#102]
Current `bin/detect` script only detects regular gradle files (`gradlew`, `build.gradle` and `settings.gradle`).
With kotlin dsl for gradle being more and more popular, it would be nice to have support for detecting `build.gradle.kts` and `settings.gradle.kts` as well.
It would require changing the `bin/detect` script just a little:
```
...
elif [ -f $1/build.gradle ]; then
  echo "Gradle" && exit 0
elif [ -f $1/settings.gradle ]; then
  echo "Gradle" && exit 0
...
```
to
```
...
elif [ -f $1/build.gradle ] || [ -f $1/build.gradle.kts ]; then
  echo "Gradle" && exit 0
elif [ -f $1/settings.gradle ] || [ -f $1/settings.gradle.kts ]; then
  echo "Gradle" && exit 0
...
```

**Comment by jaro2gw** [comment]
I forked the repo and created a [pull request](https://github.com/heroku/heroku-buildpack-gradle/pull/103)

**Comment by jaro2gw** [comment]
@Malax @edmorley I see you are the latest active maintainers of this repository, can you please review this issue and the mentioned pull request?

**Comment by sixcorners** [comment]
I don't think they even want to detect on build.gradle or settings.gradle. People seem to be pushing that wrapper thing very hard.

**Comment by Malax** [comment]
This has been fixed via @jaro2gw's PR #103 - sorry for the delay folks!
